pkg/gen/input/renovate/internal/file/renovate.json.template

```diff
@@ -15,29 +15,31 @@
       "groupName": "giantswarm modules"
     },
     {
-      "matchPackagePatterns": ["^k8s.io", "^sigs.k8s.io"],
-      "groupName": "k8s modules"
+      "matchPackagePatterns": ["^k8s.io"],
```

Comment:
This will create a single PR for all updates coming from dependencies which name starts with `k8s.io`, but only until 0.21.0